templates/payment_UI.html

```diff
@@ -0,0 +1,37 @@
+<!DOCTYPE html>
+<html lang="en">
+
+<head>
+  <meta charset="UTF-8">
+  <title>Checkout</title>
+  <link rel='stylesheet prefetch'
+	href='https://cdnjs.cloudflare.com/ajax/libs/twitter-bootstrap/4.0.0/css/bootstrap.css'>
+
+    <style>
+      span.red { color: #FF0000; }
+    </style>
+
+</head>
+
+<body>
+  <div class="container">
+    <p>
+      Dear {{data.first_name}} {{data.last_name}},
+      <br><br>
+      Thank you for registering for the July 2025 DESC Meeting at SLAC!
+      <br><br>
+      Registration fees and guest dinner tickets are billed separately.
```

Comment:
Suggest changing "Registration fees and guest dinner tickets are billed separately.  Please click the link below to proceed with these payments and complete your registration." to "To complete your registration, pay the registration fee and optionally purchase a guest dinner ticket at the link below."

Reply:
@sethdigel The change I'm about to make uses somewhat different wording than your suggestion.  Let me know if it's not acceptable.